types/src/on_chain_config/aptos_features.rs

```diff
@@ -274,6 +279,29 @@ impl Features {
     pub fn is_refundable_bytes_enabled(&self) -> bool {
         self.is_enabled(FeatureFlag::REFUNDABLE_BYTES)
     }
+
+    pub fn get_max_identifier_size(&self) -> u64 {
+        if self.is_enabled(FeatureFlag::LIMIT_MAX_IDENTIFIER_LENGTH) {
+            IDENTIFIER_SIZE_MAX
+        } else {
+            LEGACY_IDENTIFIER_SIZE_MAX
+        }
+    }
+
+    pub fn get_max_binary_format_version(&self, gas_feature_version_opt: Option<u64>) -> u32 {
```

Comment:
let's unit test this? not for now, for safeguarding future changes

Reply:
added a test to check 5 is min, 7 is max